package.json

```diff
@@ -309,12 +309,12 @@
     "browser-stdout": "1.3.1",
     "commander": "2.11.0",
     "debug": "3.1.0",
-    "diff": "3.5.0",
     "escape-string-regexp": "1.0.5",
     "glob": "7.1.2",
     "growl": "1.10.3",
     "he": "1.1.1",
     "mkdirp": "0.5.1",
+    "structured-diff": "Mingun/structured-diff",
```

Comment:
After the publication of a package I will replace the reference to a npm repository here